### ISSUE TITLE
Able to preview but can't play

### ISSUE THREAD

**Issue by chienhao10** [Namo2/InPlayerEpisodePreview#2]
Thx for creating this plugin, its just what i am looking for! but i have a small issue:

Running on Jellyfin Server (Win 10)
Installed from repo and were able to enable see the episode preview while in player.
Everything works fine expect i am not able to "play" the episode i've selected. From your short clip, i see there is a play button, but it does not show up on mine.

**Comment by Namo2** [comment]
Hello,

I am guessing you access your server trough the web page.
However the play option is currently limited to the Jellyfin Media Player Client only.

https://github.com/jellyfin/jellyfin-media-player

If you are already using the client please give me an update

**Comment by chienhao10** [comment]
> Hello,
> 
> I am guessing you access your server trough the web page. However the play option is currently limited to the Jellyfin Media Player Client only.
> 
> https://github.com/jellyfin/jellyfin-media-player
> 
> If you are already using the client please give me an update

Yes, i did run on through the webpage. I've installed Jellyfin Media Player Client as suggested, but just on the same pc as i run the server.
I see there is an update and my JMP is running the latest version of the plugin, is it necessary to upgrade the plugin on server side? Since I am keep getting error 
```
The process cannot access the file 'C:\ProgramData\Jellyfin\Server\plugins\InPlayerEpisodePreview_1.0.1.0\Namo.Plugin.InPlayerEpisodePreview.dll' because it is being used by another process.
```
not sure if this need to taken care of, or i can just ignore the plugin on server side.

Or is it safe to uninstall and reinstall on server side? hope reinstall won't lost the 'some extra data like the episode description' that was downloaded

**Comment by Namo2** [comment]
Hey, totally safe to reinstall the plugin on the server side.

To clarify this, the server side version only adds the functionality to the web page/player.
That means, the JMP Client version would also work without it.
Things like the episode descriptions are not saved locally, they will be always downloaded when you start a new episode.

**Comment by xiNeRoar** [comment]
is it possible to switch episode in web client now?

**Comment by Namo2** [comment]
Yeah that's possible now

**Comment by nfettinger** [comment]
Will you be adding support to the clients using a web page (browser version)?
